### ISSUE TITLE
Feature: People on exec mailinglist can modify event

### ISSUE THREAD

**Issue by lad6558** [DormSoup/dormsoup#20]
For example, if LSC-people send email, all LSC-exec can modify

**Comment by lad6558** [comment]
Maybe in the future when we keep track of more user settings

**Comment by gabrc52** [comment]
I don't think this is practical, unless the exec mailinglist is cc'ed, or somehow which clubs have which exec lists is written down elsewhere.
